common/templates/options.php

```diff
@@ -203,6 +205,33 @@
 </form>
 <script>
     (function ($) {
+        $.get("admin-ajax.php",
```

Comment:
Two options. Two. Not more:

~~~
FN(A, B, C)
~~~

~~~
FN(
  A,
  B,
  C
)
~~~

That's it.